bundles/org.eclipse.jface/src/org/eclipse/jface/dialogs/MessageDialog.java

```diff
@@ -246,7 +247,6 @@ private void init(String dialogTitle, Image dialogTitleImage, String dialogMessa
 		case QUESTION:
 		case QUESTION_WITH_CANCEL:
 		case CONFIRM: {
-			this.image = getQuestionImage();
```

Comment:
Maybe we should leave "CONFIRM" as is and jsut break at QUESTION/WITH_CANCEL? Or even use JFacePreferences to decide if icon should be shown or not?

Reply:
I don't see an example for confirmation dialog using a question / help icon in the guidelines https://docs.microsoft.com/en-us/windows/apps/design/controls/dialogs-and-flyouts/dialogs Check for section "Confirmation dialogs (OK/Cancel)".